### PR TITLE
create entities, repositories, services (User, Role)

### PR DESCRIPTION
**Pull request by Baldinini**

### CODE REVIEW

**Review by yurwing — APPROVED** [review]

File: src/main/java/com/example/bankservice/repository/UserRepository.java [hunk 1]
```diff
@@ -0,0 +1,9 @@
+package com.example.bankservice.repository;
+
+import com.example.bankservice.model.User;
+import java.util.Optional;
+import org.springframework.data.jpa.repository.JpaRepository;
+
+public interface UserRepository extends JpaRepository<User, Long> {
+    Optional<User> findByPhoneNumber(String phoneNumber);
```
Comment (KseniiaMakarova) [inline src/main/java/com/example/bankservice/repository/UserRepository.java]: please see checklist - we should fetch roles along with user in this method and in `findById()` as well

File: src/main/java/com/example/bankservice/service/UserService.java [hunk 2]
```diff
@@ -0,0 +1,13 @@
+package com.example.bankservice.service;
+
+import com.example.bankservice.model.User;
+
+public interface UserService {
+    void create(User user);
```
Comment (KseniiaMakarova) [inline src/main/java/com/example/bankservice/service/UserService.java]:
```suggestion
    void save(User user);
```

File: src/main/resources/application.properties [hunk 3]
```diff
@@ -1 +1 @@
-
+spring.datasource.url=jdbc:h2:mem:testdb
```
Comment (KseniiaMakarova) [inline src/main/resources/application.properties:1]: let's enable the console as well so that we could access h2 DB from the browser

**Review by KseniiaMakarova — CHANGES_REQUESTED** [review]
please add checkstyle plugin as well

**Review by KseniiaMakarova — CHANGES_REQUESTED** [review]
what about checkstyle plugin?

File: pom.xml [hunk 4]
```diff
@@ -68,6 +68,14 @@
                     </excludes>
                 </configuration>
             </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-war-plugin</artifactId>
```
Comment (KseniiaMakarova) [inline pom.xml]: is it the correct plugin?

**Review by KseniiaMakarova — CHANGES_REQUESTED** [review]

**Review by KseniiaMakarova — APPROVED** [review]

**Pull request merged**